src/Stratis.Bitcoin.Features.BlockStore/BlockRepository.cs

```diff
@@ -69,12 +69,21 @@ public interface IBlockRepository : IDisposable
         /// <param name="trxid">The transaction hash</param>
         Task<uint256> GetTrxBlockIdAsync(uint256 trxid);
 
+        /// <summary>
+        /// For every block in the database, store the transactions (or don't) according to whether TxIndex is set.
```

Comment:
Use `<see>` or English for TxIndex. I did you want to reference some existing object by its name or you want to just say some thing about transaction index.

Reply:
From the description I couldn't really understand what it is supposed to do.

Reply:
Hopefully better.